From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Otto=20Pfl=C3=BCger?= <otto.pflueger@abscue.de>
Date: Sat, 17 May 2025 20:54:33 +0200
Subject: [PATCH 106/225] dt-bindings: thermal: add compatible for Unisoc
 UMS9230
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The thermal sensor controller in UMS9230 is compatible with the one used
in UMS512.

Signed-off-by: Otto Pflüger <otto.pflueger@abscue.de>
---
 .../devicetree/bindings/thermal/sprd-thermal.yaml          | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/thermal/sprd-thermal.yaml b/Documentation/devicetree/bindings/thermal/sprd-thermal.yaml
index afa551f61..00821d188 100644
--- a/Documentation/devicetree/bindings/thermal/sprd-thermal.yaml
+++ b/Documentation/devicetree/bindings/thermal/sprd-thermal.yaml
@@ -15,7 +15,12 @@ $ref: thermal-sensor.yaml#
 
 properties:
   compatible:
-    const: sprd,ums512-thermal
+    oneOf:
+      - items:
+          - enum:
+              - sprd,ums9230-thermal
+          - const: sprd,ums512-thermal
+      - const: sprd,ums512-thermal
 
   reg:
     maxItems: 1
